site/docs/v1/tech/connectors/generic-connector.adoc

```diff
@@ -8,17 +8,80 @@ description: Connect to an arbitrary user data source over HTTP
 
 == Overview
 
+* <<Configuration>>
+* <<Using the Generic Connector>>
+* <<Security>>
+
 Generic Connectors allow you to authenticate users against or migrate them from any user datasource accessible over HTTP or TLS.
 
-image::connectors/generic-connector-create-screen.png[The Generic Connectors creation page.,width=1200,role=shadowed]
+
+=== Configuration
+
+image::connectors/generic-connector.png[The Generic Connector creation page.,width=1200,role=shadowed]
+
+=== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+An optional UUID. When this value is omitted a unique Id will be generated automatically.
+
+[field]#Name# [required]#Required#::
+A unique name to identity the identity provider. This name is for display purposes only and it can be modified later if desired.
```

Comment:
This mentions the identity provider instead of connector. If you copied it from the LDAP connector, perhaps I had a typo over there as well.

Reply:
Yup, fixed in both places. copy pasta for the win.